config/bidderinfo.go

```diff
@@ -356,6 +358,44 @@ func validateMaintainer(info *MaintainerInfo, bidderName string) error {
 	return nil
 }
 
+func validateAliasCapabilities(bidder BidderInfo, infos BidderInfos, bidderName string) error {
+	if len(bidder.AliasOf) > 0 {
+		if parentBidder, ok := infos[bidder.AliasOf]; ok {
+			if parentBidder.Capabilities == nil && bidder.Capabilities != nil {
+				return fmt.Errorf("alias capabilities should be a subset of parent bidder capabilities")
```

Comment:
You should use errors.New here. There is no advantage to using fmt.Errorf if there is no formatting in the error string.

Reply:
Can you also update tests to include coverage for this line?

Reply:
* Updated error messages to list bidders with error as part of [96e960b](https://github.com/prebid/prebid-server/pull/2996/commits/96e960b0bdee0e8670ee19308e4f2c42104939be) so keeping `fmt.Errorf`
* Added a test case as part of [1b545f5](https://github.com/prebid/prebid-server/pull/2996/commits/1b545f54db7d90138f35ccf70b7e9744301fd917)